tests/tracking/test_rest_tracking.py

```diff
@@ -223,11 +223,11 @@ def test_create_run_all_args(mlflow_client, parent_run_id_kwarg):
     experiment_id = mlflow_client.create_experiment('Run A Lot (parent_run_id=%s)'
                                                     % (parent_run_id_kwarg))
     created_run = mlflow_client.create_run(experiment_id, **create_run_kwargs)
-    run_id = created_run.info.run_uuid
+    run_id = created_run.info.run_id
     print("Run id=%s" % run_id)
 
     run = mlflow_client.get_run(run_id)
-    assert run.info.run_uuid == run_id
+    assert run.info.run_id == run_id
```

Comment:
Should we still assert on `run.info.run_uuid` here? It is a little scary to have no tests that verify that `run.info.run_uuid` is still accessible, although looking at the code change to RunInfo, it should be (i.e. the code change is pretty simple). Adding such an assert here would give us e2e coverage that `run.info.run_uuid` both against the FileStore & SQLAlchemyStore

Reply:
Open to pushback here, since we'd just delete the `run.info.run_uuid` tests eventually anyways - maybe manual testing is sufficient.